kaldb/src/main/java/com/slack/kaldb/server/RecoveryTaskCreator.java

```diff
@@ -218,6 +218,8 @@ public long determineStartingOffset(
       } else if (indexerConfig.getCreateRecoveryTasksOnStart()
           && indexerConfig.getReadFromLocationOnStart()
               == KaldbConfigs.KafkaOffsetLocation.LATEST) {
+        // todo - this appears to be able to create recovery tasks that have a start and end
```

Comment:
could we change the comment to specify this happens during next cluster creation?

Reply:
Done! Also added a detail that it seems to be worse with indexers that are created ant not immediately used.